boards/arm/pan1783_evb/Kconfig.board

```diff
@@ -0,0 +1,15 @@
+# PAN1783 EVB board configuration
+
+# Copyright (c) 2023 Panasonic Industrial Devices Europe GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NRF5340_CPUAPP_QKAA
+
+config BOARD_PAN1783_EVB_CPUAPP
+	bool "PAN1783 EVB (nRF5340) Application MCU"
+
+endif # SOC_NRF5340_CPUAPP_QKAA
```

Comment:
nit: for consistency, you could change this to:
```
config BOARD_PAN1783_EVB_CPUAPP
	bool "PAN1783 EVB (nRF5340) Application MCU"
	depends on SOC_NRF5340_CPUAPP_QKAA
```

Reply:
You're right! Thanks